packages/common/src/ContractInteractor.ts

```diff
@@ -27,26 +26,44 @@ import { address2topic, decodeRevertReason, event2topic } from './Utils'
 import {
   BaseRelayRecipientInstance,
   IForwarderInstance,
-  IKnowForwarderAddressInstance,
   IPaymasterInstance,
   IPenalizerInstance,
   IRelayHubInstance,
   IRelayRecipientInstance,
   IStakeManagerInstance
 } from '../../../types/truffle-contracts'
 
-import { Address, EventName, IntString, ObjectMap, SemVerString, Web3ProviderBaseInterface } from './types/Aliases'
+import { Address, IntString, ObjectMap, SemVerString, Web3ProviderBaseInterface } from './types/Aliases'
 import GsnTransactionDetails from './types/GsnTransactionDetails'
 
 import { Contract, TruffleContract } from './LightTruffleContract'
 import { gsnRequiredVersion, gsnRuntimeVersion } from './Version'
 import Common from 'ethereumjs-common'
 import { GSNContractsDeployment } from './GSNContractsDeployment'
-import { ActiveManagerEvents, RelayWorkersAdded, StakeInfo } from './types/GSNContractsDataTypes'
+import { StakeInfo } from './types/GSNContractsDataTypes'
 import TransactionDetails = Truffle.TransactionDetails
 
 require('source-map-support').install({ errorFormatterForce: true })
 
+type EventName = string
+
+export const CommitAdded: EventName = 'CommitAdded'
+
+export const RelayServerRegistered: EventName = 'RelayServerRegistered'
+export const RelayWorkersAdded: EventName = 'RelayWorkersAdded'
+export const TransactionRelayed: EventName = 'TransactionRelayed'
+export const TransactionRejectedByPaymaster: EventName = 'TransactionRejectedByPaymaster'
+
+const ActiveManagerEvents = [RelayServerRegistered, RelayWorkersAdded, TransactionRelayed, TransactionRejectedByPaymaster]
+
+export const HubAuthorized: EventName = 'HubAuthorized'
+export const HubUnauthorized: EventName = 'HubUnauthorized'
+export const StakeAdded: EventName = 'StakeAdded'
+export const StakeUnlocked: EventName = 'StakeUnlocked'
+export const StakeWithdrawn: EventName = 'StakeWithdrawn'
+export const StakePenalized: EventName = 'StakePenalized'
+export const OwnerSet: EventName = 'OwnerSet'
+
```

Comment:
removing.

great. this is exactly the kind of things I was afraid of. while doing the merge, it sometimes fails to recognize changes (specifically this one was automated, since there is no conflict here)

if you of other refactors you've made that would be great..